bigtable/client/table_admin.cc

```diff
@@ -125,6 +125,18 @@ bool TableAdmin::CheckConsistency(
   return consistent;
 }
 
+bool TableAdmin::WaitForConsistencyCheckImpl(
+    bigtable::TableId const& table_id,
+    bigtable::ConsistencyToken const& consistency_token) {
+  grpc::Status status;
+  bool consistent =
+      impl_.WaitForConsistencyCheckHelper(table_id, consistency_token);
+  if (not consistent) {
```

Comment:
I am not sure this is the right behavior.  Why raise an exception when the value is simply "not consistent"?  If the function was never able to talk to the server to determine the consistency state that is one thing, exceptions are the right approach then.  But if the function does manage to contact the server and the response is "not consistent" (or more accurately: "not consistent yet") then we should just return that value.